apps/studio/src/lib/import/index.js

```diff
@@ -8,38 +8,41 @@ export default class Import {
     this.options = options
     this.connection = connection
     this.table = table
-    this.sqlInserts = []
     this.logger = () => log
-  }
-
-  #sqlInserts = []
 
-  clearInsert() {
-    this.sqlInserts = []
+    this.importScriptOptions = {
+      executeOptions: { multiple: false }
+    }
   }
 
   setOptions(opt) {
     this.options = opt
-  }
-
-  addInsert(insertArr) {
-    insertArr.forEach(data => {
-      this.sqlInserts.push({
-        data: [data],
-        table: this.table.name,
-        schema: this.table.schema || null
-      })
-    })
+    this.table = opt.table
   }
   
   async buildSQL() {
-    await this.read(this.getImporterOptions({ preview: false }))
+    await this.read(this.getImporterOptions({ isPreview: false }))
     return this.connection.getImportSQL(this.sqlInserts, this.options.truncateTable)
   }
```

Comment:
We already talked about this, but just for posterity:

This doesn't seem to be used anywhere, we can maybe nuke it :)